api/v1beta1/azuremanagedcontrolplane_types.go

```diff
@@ -220,6 +232,89 @@ type AzureManagedControlPlaneSpec struct {
 	// DisableLocalAccounts disables getting static credentials for this cluster when set. Expected to only be used for AAD clusters.
 	// +optional
 	DisableLocalAccounts *bool `json:"disableLocalAccounts,omitempty"`
+
+	// SecurityProfile defines the security profile for the container service cluster.
```

Comment:
What is container service cluster?

Reply:
I think this is a copy paste from the Azure SDK types (azure sdk calls AKS "container service" for legacy reasons).

This should be 

```suggestion
	// SecurityProfile defines the security profile for cluster.
```